### DISCUSSION TITLE
Error message"Task add_mesh_generator is not registered to build MooseMesh derived objects"

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#24571] [Q&A Modules: Phase field]
Hi everyone, 

When I ran my simulation, I got this error message, why is this? How to correct it?
![image](https://github.com/idaholab/moose/assets/63679626/b7086639-debd-43e7-ad0d-f0dad3554ddf)

I also attached my input file here.
[file.txt](https://github.com/idaholab/moose/files/11629863/file.txt)

Thank you very much,
Best,

**Comment by GiudGiud** [comment]
Hello

You need to use GeneratedMeshGenerator not GeneratedMesh

Guillaume
